e2e/__tests__/abq.test.ts

```diff
@@ -0,0 +1,230 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
```

Comment:
I'm not sure what exactly we should do with the licensing headers for novel files we introduce for ABQ reasons. I don't think it's terribly important right now but maybe we want both the Facebook and an RWX attribution header? @dan-manges @TAGraves

Reply:
Yeah, the standard way to do this is to add an attribution line. It's not a problem if we miss one of these though as long as we're still releasing under the MIT license.

```
* Copyright (c) ReadWriteExecute, Inc. All Rights Reserved.
* Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
```

Reply:
I've added these to the new ABQ-specific files. I've not gone through to modify the header on files we may have touched that exist upstream.